subprojects/language-java/src/integTest/groovy/org/gradle/java/compile/incremental/AbstractSourceIncrementalCompilationIntegrationTest.groovy

```diff
@@ -510,7 +542,13 @@ sourceSets {
     }
 
     def "reports source type that does not support detection of source root"() {
+        given:
```

Comment:
Can we split this test in two? One for Groovy where we expect the message of the deprecation warning and one for Java where we expect the message why everything needed to be recompiled?

Reply:
Done, PTAL!